src/resources/ble_esp32.cc

```diff
@@ -253,6 +337,90 @@ static void blecent_on_sync(void) {
   BLEEventSource::on_started();
 }
 
+static void ble_on_reset(int reason) {
+  // Do we need this?
+}
+
+int BLEResourceGroup::init_server() {
+  if (_server_config != null) {
+    nimble_port_init();
+
+    ble_svc_gap_init();
+    ble_svc_gatt_init();
+
+    // Build the service structure
+    int service_cnt = 0;
+    for (__attribute__((unused)) BLEServerServiceResource *t: _server_config->services()) service_cnt++;
```

Comment:
I'd rather not have attributes in here. Maybe just use the `USE` macro in the loop body?

Reply:
I noticed that not many attributes in the source, I missed this when I cleaned it up. I will fix